index.html

```diff
@@ -16,6 +16,36 @@
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
   </head>
   <body>
+    <header>
+      <nav>
+        <h1>Peace and Joy</h1>
+        <div class="menu">
+          <button>Experiencing Peace and Joy</button>
+          <button>Experiencing Peace</button>
+          <button>Experiencing Joy</button>
+        </div>
+
+        <p class="description">I am a coder who loves to feel peaceful and joyful. So I am going to create a website where I can talk about things that make me feel peaceful and add joy to my life.</p>
+      </nav> 
+    </header>
+    <main>
+      <article>
+        <h2>Experiencing Peace and Joy</h2>
+        <img src="http://data.freehdw.com/bridge-over-the-green-river.jpg" alt="Lake reflecting green trees surronding the lake. There is a long wooden bridge over the lake">
+        <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Accusantium officiis ipsa rerum, deserunt nihil quod blanditiis dolorem, omnis possimus aspernatur maxime at eos ex pariatur! Things that bring me both peace and joy. Read more...</p>
+      </article>
+      <article>
+        <h2>Experiencing Peace</h2>
+        <img src="http://images.freehdw.com/800/nature-landscapes_other_wooden-house-in-forest_3295.jpg" alt="Small house in a forest. Morning sunrays falling over the trees and top of the house."
```

Comment:
can we download the image and link to it with a relative path `<img src="images/picture.jpg">`